server/build-plugins.js

```diff
@@ -1,15 +1,18 @@
 'use strict';
 
 const fs = require('fs');
-const os = require('os');
+
 const assert = require('assert');
 const { resolve } = require('path');
+const Config = require('bcfg');
 const { format } = require('prettier');
 const { execSync } = require('child_process');
 
 const logger = require('./logger');
 
-const pluginsConfig = resolve(os.homedir(), '.bpanel/config.js');
+const config = new Config('bpanel');
```

Comment:
Nice thing about using `bcfg` is that we can simply rely on the single use of `os.homedir()` that uses for the default `prefix` :) 

Using this for the bpanel config (in addition to the bpanel clients) is experimental. It's a little hacky because I have to manually pass it around in a few places given how the builds work, but it should hopefully allow for a little more predictability and easier to maintain.